__tests__/app.test.js

```diff
@@ -7,38 +7,44 @@ const mongoose = require('mongoose');
 const Recipe = require('../lib/models/Recipe');
 
 describe('app routes', () => {
+  // connect to database
   beforeAll(() => {
     connect();
   });
-
+  // drop database connection
   beforeEach(() => {
     return mongoose.connection.dropDatabase();
   });
-
+  // close database connection
   afterAll(() => {
     return mongoose.connection.close();
   });
 
```

Comment:
Everything looks really clean and organized! One suggestion to make it slightly cleaner is when you are writing your routes and route tests, order them in the same way, for example:
if you wrote your routes in this order post, get, getById, patch, delete then have your tests reflect the same order!